ci_build/azure_pipelines/templates/keras2onnx_application_tests.yml

```diff
@@ -21,10 +21,12 @@ steps:
       pip install $(ONNX_PATH)
       pip install h5py==2.9.0
       pip install parameterized
+      pip install numpy==1.21.5 flatbuffers==2.0
+      $(INSTALL_ORT)
```

Comment:
Finished, some keras apis issue will be fixed in https://github.com/onnx/tensorflow-onnx/pull/1888.